dali/operators/signal/decibel/to_decibels_op_cpu.cc

```diff
@@ -20,24 +20,31 @@
 namespace dali {
 
 DALI_SCHEMA(ToDecibels)
-    .DocStr(R"code(Converts a magnitude (real, positive) to the decibel scale, according to the
-formula::
+    .DocStr(R"code(Converts a magnitude (real, positive) to the decibel scale by using
+the following formula::
 
   min_ratio = pow(10, cutoff_db / multiplier)
   out[i] = multiplier * log10( max(min_ratio, input[i] / reference) ))code")
     .NumInput(kNumInputs)
     .NumOutput(kNumOutputs)
     .AddOptionalArg("multiplier",
-      R"code(Factor by which we multiply the logarithm (typically 10.0 or 20.0 depending if we
-are dealing with a squared magnitude or not).)code",
+      R"code(Factor by which the logarithm is multiplied, typically 10.0 or 20.0, and
```

Comment:
Now the new sentence means something different. I suggest:
"is multiplied. The value is typically 10.0 or 20.0, which depends on whether the magnitude is squared."

Reply:
Done